server.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/nyaruka/gocommon/jsonx"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	"golang.org/x/exp/slog"
```

Comment:
"log/slog"

Reply:
replaced properly

Reply:
the `golang.org/x/exp` is where new experimental features are tried out before sometimes becoming part of the golang std lib - so `slog` first existed there before being added to go 1.21